apache/Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM fedora
+FROM docker.io/fedora
 MAINTAINER http://fedoraproject.org/wiki/Cloud
 
+LABEL RUN='docker run -d -p 80:80 $IMAGE'
```

Comment:
Some people may not be familiar with what these labels are for.  Please add a comment here, and update the README. This one in particular, it's our most downloaded image.